.github/workflows/ci.yml

```diff
@@ -0,0 +1,68 @@
+on: [push, pull_request]
+
+name: Continuous integration
+
+jobs:
+  check:
+    name: Check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+      - uses: actions-rs/cargo@v1
+        with:
+          command: check
```

Comment:
If we run tests, we don't need a separate `check` step, since testing is a superset of `check`.

Reply:
I removed it.